src/main/java/team03/airdnb/user/UserService.java

```diff
@@ -0,0 +1,58 @@
+package team03.airdnb.user;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import team03.airdnb.accommodation.Accommodation;
+import team03.airdnb.accommodation.dto.response.AccommodationListDto;
+import team03.airdnb.favorite.Favorite;
+import team03.airdnb.reservation.dto.response.ReservationShowDto;
+import team03.airdnb.review.Review;
+import team03.airdnb.review.dto.response.ReviewShowDto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public List<ReservationShowDto> showReservations(Long userId) {
+        User user = userRepository.findById(userId).get();
+
+        return user.getReservations().stream()
+                .map(ReservationShowDto::of)
+                .collect(Collectors.toList());
+    }
+
+    public List<AccommodationListDto> showFavorites(Long userId) {
+        User user = userRepository.findById(userId).get();
+        List<Favorite> favorites = user.getFavorites();
+
+        List<AccommodationListDto> accommodationListDtos = new ArrayList<>();
+
+        for (Favorite favorite : favorites) {
+            Accommodation accommodation = favorite.getAccommodation();
+            AccommodationListDto accommodationListDto = AccommodationListDto.of(accommodation, accommodation.getAccommodationAmenities());
+            accommodationListDtos.add(accommodationListDto);
+        }
+
+        return accommodationListDtos;
+    }
+
+    public List<ReviewShowDto> showReviews(Long userId) {
+        User user = userRepository.findById(userId).get();
+        List<Review> reviews = user.getReviews();
+
+        List<ReviewShowDto> reviewShowDtos = new ArrayList<>();
```

Comment:
```suggestion
        List<ReviewShowDto> reviewShowDtos = reviews.stream()
        .map(ReviewShowDto::of)
        .collect(Collectors.toList());
```

Stream API를 사용하면 위와 같이 개선할 수도 있습니다!
단순 맵핑의 경우 Stream API를 사용하면 가독성이 좋아지는 경우가 있어서 공부해보시는 것도 추천드려요!